README.md

```diff
@@ -1,3 +1,11 @@
+**Note**:
+
+This branch (main) is for the latest V6 version of the TS/JS SDK generator - [@autorest/typescript](https://www.npmjs.com/package/@autorest/typescript).

```

Comment:
We probably should not call it V6 anymore. cc @joheredi

Reply:
I think we should refer to them now using their names on npm, `@autorest/typescript` and `@microsoft.azure/autorest.typescript`

Reply:
@ramya-rao-a what do you think?

Reply:
What is the approach used by the code generators in other languages? Usually in other feature crews, we start with the last major version + 1 even if the package name is different.

Reply:
@ramya-rao-a to give you more context, we as the crew, already committed to have v6-branded version released as v1.0.0-beta.1. I know Python used the approach of last major version + 1. @sarangan12 did some research on what the other languages did but I do not remember the findings tbh.

Now, since we already committed to v1 release, I feel like it is very confusing to still call it v6. I think the following naming proposal might work better than v6:
- `@autorest/typescript` vs `@microsoft.azure/autorest.typescript`
- New code gen vs legacy code gen
- v1 vs v4 (least favorite because it is still confusing IMO)

Reply:
1. Removed the reference to V6 and V4 Versions. 
2. As for the version number, we have discussed this earlier with autorest crew and decided to start with 1.0.0-beta.x version for the SDK Generator.